src/fhirtypes/StructureDefinition.ts

```diff
@@ -83,6 +88,13 @@ export class StructureDefinition {
     return this._baseStructureDefinition;
   }
 
+  /**
+   * Get the Structure Definition for Structure Definition
+   */
+  getStructureDefinitionStructureDefinition() {
```

Comment:
Should this maybe be `private`?  Is there a good reason to expose it?

Reply:
Also, if you made this take a `resolve` function as an argument, then it could essentially do something like:
```
if (this._structureDefinitionStructureDefinition == null) {
  this._structureDefinitionStructureDefinition = resolve('StructureDefinition');
}
return this._structureDefinitionStructureDefinition;
```
This allows dynamic resolution of the SD when needed, but caches it after that.

Reply:
Oh nice this is much cleaner.